{
  "Faults": [
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-01T16:18:00.000+0000",
      "latitude": 51.46517590768635,
      "estimatedRestoration": "2023-12-02T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6443'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6443",
      "reference": "MX6443",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-12-01T17:45:00.000+0000",
      "affectedCustomerCount": 14,
      "name": "TW19 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "TW19",
        "TW19 6AG",
        "TW19 6BD",
        "TW19 6BF",
        "TW19 6BG",
        "TW19 6BJ",
        "TW19 6BL",
        "TW19 6BN",
        "TW19 6BQ",
        "TW19 6DA"
      ],
      "location": {
        "coordinates": [
          -0.5016335206869315,
          51.46517590768635
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-12-01T17:30:00.000+0000",
      "UUID": "f26aa4e0-6b1b-4c8e-84ac-792e00af4031",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.5016335206869315,
      "resolved": false
    },
    {
      "latitude": 51.38566438113576,
      "type": "LV",
      "reference": "MX6624",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-12-01T21:45:00.000+0000",
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "c9aaccd5-56ef-48f2-a73c-861fc93717ed",
      "minLongitude": -0.9171,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9101991709372911,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-01T20:09:00.000+0000",
      "estimatedRestoration": "2023-12-02T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6624'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6624",
      "maxLongitude": -0.9033,
      "minLatitude": 51.3833,
      "name": "RG2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG2 9LZ",
        "RG2 9PN",
        "RG2 9PP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9171,
              51.39
            ],
            [
              -0.9033,
              51.39
            ],
            [
              -0.9033,
              51.3833
            ],
            [
              -0.9171,
              51.3833
            ],
            [
              -0.9171,
              51.39
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-01T21:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.39,
      "etrUpdate": true
    },
    {
      "latitude": 51.598241593749755,
      "type": "LV",
      "reference": "MX6737",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-12-02T04:45:00.000+0000",
      "affectedCustomerCount": 13,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "dfc0e740-e86b-41e7-bf65-3b2d91f872b4",
      "minLongitude": -1.3406,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3366308391085548,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T04:45:00.000+0000",
      "estimatedRestoration": "2023-12-02T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6737'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6737",
      "maxLongitude": -1.3247,
      "minLatitude": 51.5951,
      "name": "OX12 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX12 8HU",
        "OX12 8JG",
        "OX12 8JJ",
        "OX12 8JL",
        "OX12 8JW",
        "OX12 8LQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3406,
              51.6007
            ],
            [
              -1.3247,
              51.6007
            ],
            [
              -1.3247,
              51.5951
            ],
            [
              -1.3406,
              51.5951
            ],
            [
              -1.3406,
              51.6007
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T04:20:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.6007,
      "etrUpdate": true
    },
    {
      "latitude": 50.73753528389917,
      "type": "PSI",
      "reference": "221567",
      "etaUpdate": true,
      "affectedCustomerCount": 19,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "504f807a-c058-4e81-b9ef-00c439469a34",
      "minLongitude": -2.5696,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.535513242189985,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T08:30:00.000+0000",
      "estimatedRestoration": "2023-12-02T16:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '221567'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/221567",
      "maxLongitude": -2.5166,
      "minLatitude": 50.722,
      "name": "DT2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT2 9DZ",
        "DT2 9ET",
        "DT2 9NJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.5696,
              50.75
            ],
            [
              -2.5166,
              50.75
            ],
            [
              -2.5166,
              50.722
            ],
            [
              -2.5696,
              50.722
            ],
            [
              -2.5696,
              50.75
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.75,
      "etrUpdate": true
    },
    {
      "latitude": 51.26556162171561,
      "type": "LV",
      "reference": "MX6872",
      "etaUpdate": true,
      "affectedCustomerCount": 4,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5beb0e2e-af77-41cf-b384-75fb42aa85cc",
      "minLongitude": -0.9998,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9940962187647907,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T08:31:00.000+0000",
      "estimatedRestoration": "2023-12-02T13:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6872'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6872",
      "maxLongitude": -0.9824,
      "minLatitude": 51.262,
      "name": "RG27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG27 9PT",
        "RG27 9PZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9998,
              51.2683
            ],
            [
              -0.9824,
              51.2683
            ],
            [
              -0.9824,
              51.262
            ],
            [
              -0.9998,
              51.262
            ],
            [
              -0.9998,
              51.2683
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T09:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2683,
      "etrUpdate": true
    },
    {
      "latitude": 51.756629697784014,
      "type": "LV",
      "reference": "MX6926",
      "etaUpdate": true,
      "affectedCustomerCount": 5,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "091b5df1-33a5-4adc-9ff7-2826e7bbc124",
      "minLongitude": -1.2535,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2508050952751855,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T09:39:00.000+0000",
      "estimatedRestoration": "2023-12-02T13:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6926'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6926",
      "maxLongitude": -1.2484,
      "minLatitude": 51.7552,
      "name": "OX1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX1 3SZ",
        "OX1 3TA",
        "OX1 3TD",
        "OX1 3TR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2535,
              51.759
            ],
            [
              -1.2484,
              51.759
            ],
            [
              -1.2484,
              51.7552
            ],
            [
              -1.2535,
              51.7552
            ],
            [
              -1.2535,
              51.759
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T10:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.759,
      "etrUpdate": true
    },
    {
      "latitude": 51.50970020405087,
      "type": "LV",
      "reference": "MX6953",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "5938d591-7312-40d1-8222-5612a01cf94d",
      "minLongitude": -0.6234,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6214787552743875,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T10:34:00.000+0000",
      "estimatedRestoration": "2023-12-02T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6953'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6953",
      "maxLongitude": -0.6187,
      "minLatitude": 51.5083,
      "name": "SL1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL1 2YR",
        "SL1 2YW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6234,
              51.5102
            ],
            [
              -0.6187,
              51.5102
            ],
            [
              -0.6187,
              51.5083
            ],
            [
              -0.6234,
              51.5083
            ],
            [
              -0.6234,
              51.5102
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T11:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5102,
      "etrUpdate": true
    },
    {
      "latitude": 51.50635143036553,
      "type": "HV",
      "reference": "MX6977",
      "etaUpdate": true,
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "9d27b8d6-2ce6-47fa-8ccf-fe8497f6d94c",
      "minLongitude": -1.1451,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1343021406216325,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T10:54:00.000+0000",
      "estimatedRestoration": "2023-12-02T15:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6977'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6977",
      "maxLongitude": -1.1222,
      "minLatitude": 51.4971,
      "name": "RG8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG8 9ND",
        "RG8 9NE",
        "RG8 9PA",
        "RG8 9SY",
        "RG8 9TA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1451,
              51.5135
            ],
            [
              -1.1222,
              51.5135
            ],
            [
              -1.1222,
              51.4971
            ],
            [
              -1.1451,
              51.4971
            ],
            [
              -1.1451,
              51.5135
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5135,
      "etrUpdate": true
    },
    {
      "latitude": 51.96120671776542,
      "type": "LV",
      "reference": "MX7016",
      "etaUpdate": true,
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "db211d53-8efb-40e1-aca1-da381867bb81",
      "minLongitude": -1.4275,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4238503462343681,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T10:57:00.000+0000",
      "estimatedRestoration": "2023-12-02T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7016'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7016",
      "maxLongitude": -1.4166,
      "minLatitude": 51.9587,
      "name": "OX7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX7",
        "OX7 4AL",
        "OX7 4AN",
        "OX7 4DB",
        "OX7 4DD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4275,
              51.964
            ],
            [
              -1.4166,
              51.964
            ],
            [
              -1.4166,
              51.9587
            ],
            [
              -1.4275,
              51.9587
            ],
            [
              -1.4275,
              51.964
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T12:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.964,
      "etrUpdate": true
    },
    {
      "latitude": 50.73512281821548,
      "type": "LV",
      "reference": "MX7087",
      "etaUpdate": true,
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ba6f97be-bc52-4bf2-80db-7d6a777c6dc1",
      "minLongitude": -1.9595,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9531863282698636,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T11:56:00.000+0000",
      "estimatedRestoration": "2023-12-02T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7087'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7087",
      "maxLongitude": -1.9481,
      "minLatitude": 50.7335,
      "name": "BH15 & BH14 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "BH12 3LH",
        "BH14 0RS",
        "BH14 0RT",
        "BH14 0RY",
        "BH15 3NZ",
        "BH15 3PA"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9595,
              50.7379
            ],
            [
              -1.9481,
              50.7379
            ],
            [
              -1.9481,
              50.7335
            ],
            [
              -1.9595,
              50.7335
            ],
            [
              -1.9595,
              50.7379
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7379,
      "etrUpdate": true
    },
    {
      "latitude": 51.50676053770684,
      "type": "LV",
      "reference": "MX7109",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d3453b1c-63f2-4afb-b320-9adbdf0de0b0",
      "minLongitude": -0.419,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.41645010482436706,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T12:03:00.000+0000",
      "estimatedRestoration": "2023-12-02T17:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7109'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7109",
      "maxLongitude": -0.4081,
      "minLatitude": 51.5029,
      "name": "UB3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB3 3BT",
        "UB3 3EL",
        "UB3 3EP",
        "UB3 3ER",
        "UB3 3EW",
        "UB3 3NB",
        "UB3 4BD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.419,
              51.5085
            ],
            [
              -0.4081,
              51.5085
            ],
            [
              -0.4081,
              51.5029
            ],
            [
              -0.419,
              51.5029
            ],
            [
              -0.419,
              51.5085
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T13:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "bb914359-a402-4e3e-9730-691be457ca96"
      ],
      "maxLatitude": 51.5085,
      "etrUpdate": true
    },
    {
      "latitude": 51.021063715795535,
      "type": "LV",
      "reference": "MX7142",
      "etaUpdate": true,
      "affectedCustomerCount": 14,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1d3edeae-a4e9-465e-88b4-0e9719caf01f",
      "minLongitude": -1.3374,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2549621934292836,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T12:28:00.000+0000",
      "estimatedRestoration": "2023-12-02T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7142'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7142",
      "maxLongitude": -1.0129,
      "minLatitude": 50.8604,
      "name": "SO22 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO9 3QW",
        "SO22 5JE",
        "SO22 6EW",
        "SO22 6RD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3374,
              51.0744
            ],
            [
              -1.0129,
              51.0744
            ],
            [
              -1.0129,
              50.8604
            ],
            [
              -1.3374,
              50.8604
            ],
            [
              -1.3374,
              51.0744
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0744,
      "etrUpdate": true
    },
    {
      "latitude": 51.072916769444156,
      "type": "LV",
      "reference": "MX7140",
      "etaUpdate": true,
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "45c471f5-c1fe-4fa8-8384-e6d0ff705eaa",
      "minLongitude": -1.3367,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.334517836226952,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T12:20:00.000+0000",
      "estimatedRestoration": "2023-12-02T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7140'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7140",
      "maxLongitude": -1.3312,
      "minLatitude": 51.0717,
      "name": "SO22 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO22 5HY",
        "SO22 5JE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3367,
              51.0739
            ],
            [
              -1.3312,
              51.0739
            ],
            [
              -1.3312,
              51.0717
            ],
            [
              -1.3367,
              51.0717
            ],
            [
              -1.3367,
              51.0739
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.0739,
      "etrUpdate": true
    },
    {
      "latitude": 57.7605433217654,
      "type": "LV",
      "reference": "MX6871",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "7d74c186-6581-4328-b5ae-b0c062bd5850",
      "minLongitude": -5.0549,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.024201923886467,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T08:31:00.000+0000",
      "estimatedRestoration": "2023-12-02T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX6871'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX6871",
      "maxLongitude": -4.7335,
      "minLatitude": 57.6979,
      "name": "IV23 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV23 2PJ",
        "IV23 2RX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.0549,
              57.8516
            ],
            [
              -4.7335,
              57.8516
            ],
            [
              -4.7335,
              57.6979
            ],
            [
              -5.0549,
              57.6979
            ],
            [
              -5.0549,
              57.8516
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-12-02T08:35:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.8516,
      "etrUpdate": true
    },
    {
      "latitude": 58.42610488153149,
      "type": "PSI",
      "reference": "221654",
      "etaUpdate": true,
      "affectedCustomerCount": 8,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "69e1431f-2729-4668-b468-2f64a1aec014",
      "minLongitude": -5.0545,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.992536291471302,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T10:00:00.000+0000",
      "estimatedRestoration": "2023-12-02T14:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '221654'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/221654",
      "maxLongitude": -4.3917,
      "minLatitude": 58.023,
      "name": "IV27 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV27 4RA",
        "IV27 4RL",
        "IV27 4RN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -5.0545,
              58.4499
            ],
            [
              -4.3917,
              58.4499
            ],
            [
              -4.3917,
              58.023
            ],
            [
              -5.0545,
              58.023
            ],
            [
              -5.0545,
              58.4499
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 58.4499,
      "etrUpdate": true
    },
    {
      "latitude": 56.56428365589199,
      "type": "LV",
      "reference": "MX7051",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "5f7b2334-c3d1-4014-accd-10776e4372de",
      "minLongitude": -2.6143,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.611378489550218,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-12-02T11:24:00.000+0000",
      "estimatedRestoration": "2023-12-02T14:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MX7051'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MX7051",
      "maxLongitude": -2.6082,
      "minLatitude": 56.5621,
      "name": "DD11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD11 3ES",
        "DD11 3EU",
        "DD11 3EX",
        "DD11 3EY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.6143,
              56.5665
            ],
            [
              -2.6082,
              56.5665
            ],
            [
              -2.6082,
              56.5621
            ],
            [
              -2.6143,
              56.5621
            ],
            [
              -2.6143,
              56.5665
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.5665,
      "etrUpdate": true
    }
  ]
}
